driver/src/orderbook/streamed/updating_orderbook.rs

```diff
@@ -64,6 +76,7 @@ impl UpdatingOrderbook {
             .past_events(BlockNumber::Number(from_block.into()), to_block)
             .await?;
         self.handle_events(context, events, from_block).await?;
+        context.orderbook.write_to_file(&self.filestore)?;
```

Comment:
It might be better log this error if it occurs but not make the whole function fail. So instead of `?` I suggest `if let Err(err) = ... { log error }`. 
This way if we for some reason cannot write to the file at least the orderbook itself still works, it just doesn't update.